cloudflare/src/endpoints/workers/create_tail.rs

```diff
@@ -25,12 +29,16 @@ impl<'a> Endpoint<WorkersTail, (), CreateTailParams> for CreateTail<'a> {
         )
     }
     fn body(&self) -> Option<CreateTailParams> {
-        Some(self.params.clone())
+        if self.params.url.is_some() {
+            Some(self.params.clone())
+        } else {
+            None
+        }
     }
 }
 
 #[derive(Serialize, Clone, Debug)]
 pub struct CreateTailParams {
```

Comment:
I suggest this object should now derive Default, because its only field includes Default.